### PR TITLE
smartcontract: print proper error message on failed testinvoke

### PR DESCRIPTION
**Pull request by fyrchik**
```
> neo-go contract testinvokefunction 152d147a02525ccfa46ed98e2c983bb70f0472e1 notexist -r http://172.21.0.2:30333
Warning: FAULT VM state returned from the RPC node: at instruction 36 (SYSCALL): failed to invoke syscall 1381727586: method not found: notexist/0
. Use --force flag to save the transaction anyway.
```

1. Do not print info about `--force` flag if we don't send or save a transaction.
2. Do not print dots on a new line.

Signed-off-by: Evgeniy Stratonikov <evgeniy@nspcc.ru>

### CODE REVIEW

**Review by roman-khimov — APPROVED** [review]

File: cli/smartcontract/smart_contract.go [hunk 1]
```diff
@@ -681,7 +681,7 @@ func invokeWithArgs(ctx *cli.Context, acc *wallet.Account, wall *wallet.Wallet,
 			}
 		}
 		if !ctx.Bool("force") {
-			return sender, cli.NewExitError(errText+". Use --force flag to "+action+" the transaction anyway.", 1)
+			return sender, cli.NewExitError(errText+".\nUse --force flag to "+action+" the transaction anyway.", 1)
 		}
 		fmt.Fprintln(ctx.App.Writer, errText+". "+process+" transaction...")
```
Comment (AnnaShaleva) [inline cli/smartcontract/smart_contract.go:686]: Let's add newline after `.` also.

Comment (fyrchik) [reply cli/smartcontract/smart_contract.go:686]: Do you mean the last dot? Newline will be printed on exit anyway.

Comment (AnnaShaleva) [reply cli/smartcontract/smart_contract.go:686]: No, I mean dot in the middle of the `fmt.Fprintln(ctx.App.Writer, errText+". "+process+" transaction...")` line, the case when there's a `--force` flag and we're sending the transaction anyway.

Comment (AnnaShaleva) [reply cli/smartcontract/smart_contract.go:686]: Just for the consistency between L684 and L686.

**Review by AnnaShaleva — COMMENTED** [review]
Also, CLI test need to be fixed.

File: cli/smartcontract/smart_contract.go (hunk 1 above)
Comment (fyrchik) [reply cli/smartcontract/smart_contract.go]: Uh, right, done!

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**